include/ginkgo/core/matrix/dense.hpp

```diff
@@ -441,6 +441,49 @@ class Dense
         return create_submatrix(rows, columns, this->get_stride());
     }
 
+    /*
+     * Create a real view of the (potentially) complex original matrix.
+     */
+    std::unique_ptr<Dense<remove_complex<ValueType>>> view_as_real()
```

Comment:
Just the obligatory naming discussion: Since we already have a `create_submatrix`, how about naming this `create_real_view` with a more "imperative" base?